mlflow/server/js/src/components/ParallelCoordinatesPlotPanel.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import ParallelCoordinatesPlotView from './ParallelCoordinatesPlotView';
+import { ParallelCoordinatesPlotControls } from './ParallelCoordinatesPlotControls';
+import { getSharedMetricKeysByRunUuids, getSharedParamKeysByRunUuids } from '../reducers/Reducers';
+import _ from 'lodash';
+
+import './ParallelCoordinatesPlotPanel.css';
+
+export class ParallelCoordinatesPlotPanel extends React.Component {
+  static propTypes = {
+    runUuids: PropTypes.arrayOf(String).isRequired,
+    sharedParamKeys: PropTypes.arrayOf(String).isRequired,
+    sharedMetricKeys: PropTypes.arrayOf(String).isRequired,
+  };
+
+  static defaultProps = {
+    sharedParamKeys: [],
+    sharedMetricKeys: [],
+  };
+
+  state = {
+    selectedParamKeys: this.props.sharedParamKeys, // default select all parameters
+    selectedMetricKeys: this.props.sharedMetricKeys.slice(0, 1),
```

Comment:
Can you add a comment about what happens if `sharedMetricKeys` is empty array?

Reply:
There will be no color scaling if no metric is selected. Will comment that.